File: test/interface/golden/__tests__/product/Card_spec.ml
let () =
  AesonSpec.sampleGoldenAndServerSpec Card.decodeSuit Card.encodeSuit "suit" "http://localhost:8081/suit" "__tests__/golden/product/Suit.json";

  AesonSpec.sampleGoldenAndServerSpec Card.decodeCard Card.encodeCard "card" "http://localhost:8081/card" "__tests__/golden/product/Card.json";

File: test/interface/golden/lib/js/__tests__/product/Card_spec.js
'use strict';

var Card      = require("../../product/Card.js");
var AesonSpec = require("bs-aeson-spec/lib/js/src/AesonSpec.js");

AesonSpec.sampleGoldenAndServerSpec(Card.decodeSuit, Card.encodeSuit, "suit", "http://localhost:8081/Card/Suit", "golden/product/Suit");

AesonSpec.sampleGoldenAndServerSpec(Card.decodeCard, Card.encodeCard, "card", "http://localhost:8081/Card/Card", "golden/product/Card");

/*  Not a pure module */
